packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -465,6 +468,7 @@ export const defaultDDSFuzzSuiteOptions: DDSFuzzSuiteOptions = {
 	detachedStartOptions: {
 		attachProbability: 0.2,
 		enabled: true,
+		numOpsBeforeAttach: 0,
```

Comment:
i think the default should be more than 0, as with the previous mechanism, attachProbability, we would do some